salt/metalk8s/salt/master/files/master-99-metalk8s.conf.j2

```diff
@@ -45,7 +45,9 @@ external_auth:
     storage-operator:
       - '*':
         - 'disk.dump'
-        - 'state.sls'
+        - 'state.sls':
```

Comment:
Ho, this was simpler than I thought ^^

Reply:
That's the theory, not working atm :sweat_smile:

Reply:
OK it looks like it's working actually!